Mergin/projects_manager.py

```diff
@@ -86,6 +87,22 @@ def create_project(self, project_name, project_dir, is_public, namespace):
         QApplication.setOverrideCursor(Qt.WaitCursor)
         try:
             self.mc.create_project(full_project_name, is_public)
+        except ClientError as e:
+            QApplication.restoreOverrideCursor()
+            msg = str(e)
+            # User friendly error messages
+            if e.http_error == 409:
+                msg = f'Project with the name "{project_name}" already exists in the workspace "{namespace}"\n Try renaming the project'
+            elif e.http_error == 422 and e.server_code == ErrorCode.ProjectsLimitHit.value:
```

Comment:
Checking only the error code should be enough and it is also better for the future if we'd like to change the http response code.

```suggestion
            elif e.server_code == ErrorCode.ProjectsLimitHit.value:
```

Reply:
Fixed for this one and all others occurrences in latest commit